site/src/components/page-faq/page-faq.tsx

```diff
@@ -12,55 +12,53 @@ export class PageFaq {
     * see: FaqData.ts
     * see: app-root.tsx
     */
-   @Prop() public data?: {sectionTitle: string,  questions: { question: string, answer: () => string }[]}[];
+   @Prop() public data?: { sectionTitle: string, questions: { question: string, answer: () => string }[] }[];
 
    /**
     * The page's title
     */
-   @Prop() public page_title?: string;
+   @Prop() public pageTitle?: string;
 
    @State() private isModalOpen: boolean = false;
 
    public render() {
       const data = this.data || [];
-      const isModalOpen = this.isModalOpen;
+      const { isModalOpen } = this;
 
       return (
-      <Host>
-         <h1>{ this.page_title || "" }</h1>
+         <Host>
+            <h1>{this.pageTitle || ""}</h1>
```

Comment:
Is my linting messed up?

Reply:
Yeah it seems there are some things it doesn't fix automatically? Variable renaming seems to be one of them. Maybe that's documented somewhere, dunno